### ISSUE TITLE
Update to Alpha 15

### ISSUE THREAD

**Issue by DrKrieger1999** [simwir/Shuttle-Train#39]
Can you patch the mod to run with Factorio 0.15 released today? # @

**Comment by folknor** [comment]
Hi, I've uploaded a remake of shuttle trains to https://mods.factorio.com/mods/folk/folk-shuttle
It could use some testing.
@simwir please let me know if you want me to remove it, or otherwise talk about it.

**Comment by simwir** [comment]
@vhaarr Sorry I have been so hard to get a hold off, you have my blessing to release an updated version of the mod. A mention that is stems from my mod would be nice. @Nexela has also expressed his desire to help keep the mod updated and I am thinking of making the two of you contributers on this github. That way you can use this as the place to have the code for the mod, but still be able to update it. If you would like to discuss further please reply here or email me at contact@simwir.dk

**Comment by Nexela** [comment]
Yeah Folk beat me to it, So I hope he understands he has to implement the features I ask for now :)

**Comment by folknor** [comment]
Thank you. Your name is certainly prominent on the new mod portal entry and info.json, there should be absolutely no question that your mod is the original, and that mine is a remake.

I don't know how to respond to the github contributor talk without sounding overly negative, but:
1. I have no need for hosting code on github or being a contributor to this repository.
2. I also don't mind it.

Both statements are true, and are neither negative nor positive. There is no hidden meaning in either.
If you want me to do anything in particular with github, I will do what you will.
If not, I will not do anything.

And yes, if you have any requests then please just let me know :-)

**Comment by simwir** [comment]
I will not impose any coding style on you, and if you are willing to use the time to keep the mod alive I will be grateful. When I get a moment I will update the mod portal to link to your version for an updated version
